azkaban-common/src/main/java/azkaban/executor/container/ContainerCleanupManager.java

```diff
@@ -116,6 +121,17 @@ public void cleanUpStaleFlows() {
         Entry::getKey).forEach(this::cleanUpStaleFlows);
   }
 
+  /**
+   * Try to clean the stale containers that are older than maximum azkaban flow running time
+   */
+  public void cleanUpStaleContainers() {
+    try {
+      this.containerizedImpl.deleteAgedContainers(validityMap.get(Status.RUNNING).getFirst());
```

Comment:
Looks like the way it's currently implemented, any unchecked exception while deleting a container will prevent the deletion of other 'stale' containers. 
Please handle all exceptions (including checked) within `deleteAgedContainers` and/or other underlying methods. 
Note that this is also good use-case for suppressed exceptions: https://www.baeldung.com/java-suppressed-exceptions
(Not necessary in our case as long as we log all the unchecked exceptions)

Reply:
It's now handled in deleteAgedContainers